### PR TITLE
update katib doc

### LINKED ISSUE #250: Katib docs need to be updated to use the StudyJob CRD
The docs for Katib should be updated to use the StudyJob CRD.

### PR DESCRIPTION
**Pull request by YujiOshima**
Fix #250 
Signed-off-by: YujiOshima <yuji.oshima0x3fd@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/263)

### CODE REVIEW

File: content/docs/guides/components/hyperparameter.md [hunk 1]
```diff
@@ -34,13 +34,110 @@ kubectl get pod -n kubeflow  # Find your vizier-core and modedb-frontend pods
...
+## Creating a Study Job
+You can create Study Job for Katib by defining a StudyJob config file.
+See [random algorithm example](https://github.com/kubeflow/katib/blob/master/examples/random-example.yaml).
 
-Run the client [script](https://github.com/kubeflow/katib/blob/master/examples/client-example.go)
 ```
-go run client-example.go
+$ kubectl apply -f random-example.yaml
+```
+Only this command, a study will start, generate hyper-parameters and save the results.
```
Comment (jlewi) [inline content/docs/guides/components/hyperparameter.md]: Only this command, a study will start, generate hyper-parameters and save the results. -> 
"Running this command will launch a StudyJob. The study job will run a series of training jobs to train models using different hyperparameters and save the results."

File: content/docs/guides/components/hyperparameter.md [hunk 2]
```diff
@@ -34,13 +34,110 @@ kubectl get pod -n kubeflow  # Find your vizier-core and modedb-frontend pods
...
+The configurations for the study(hyper-parameter feasible space, optimization parameter, optimization goal, suggestion algorithm, and so on) are defined in `random-example.yaml`,
```
Comment (r2d4) [inline content/docs/guides/components/hyperparameter.md]: add space `study (`

File: content/docs/guides/components/hyperparameter.md [hunk 3]
```diff
@@ -34,13 +34,110 @@ kubectl get pod -n kubeflow  # Find your vizier-core and modedb-frontend pods
...
+In this demo, hyper-parameters are embbeded as args.
+You can embbed in another way(e.g. eviroment values) by using template.
```
Comment (r2d4) [inline content/docs/guides/components/hyperparameter.md]: add space `way (`

Comment (r2d4) [reply content/docs/guides/components/hyperparameter.md]: typo `embbed`->`embed`, `eviromnet`->`environment`

Comment (r2d4) [reply content/docs/guides/components/hyperparameter.md]: `by using _this_ template`

File: content/docs/guides/components/hyperparameter.md [hunk 4]
```diff
@@ -34,13 +34,110 @@ kubectl get pod -n kubeflow  # Find your vizier-core and modedb-frontend pods
...
+See [random algorithm example](https://github.com/kubeflow/katib/blob/master/examples/random-example.yaml).
 
-Run the client [script](https://github.com/kubeflow/katib/blob/master/examples/client-example.go)
 ```
-go run client-example.go
+$ kubectl apply -f random-example.yaml
+```
+Only this command, a study will start, generate hyper-parameters and save the results.
+The configurations for the study(hyper-parameter feasible space, optimization parameter, optimization goal, suggestion algorithm, and so on) are defined in `random-example.yaml`,
+In this demo, hyper-parameters are embbeded as args.
```
Comment (r2d4) [inline content/docs/guides/components/hyperparameter.md]: embedded

**Comment by r2d4** [comment]
/approve
/lgtm

Thank you!

**Pull request merged**